.github/scripts/check-commit.sh

```diff
@@ -12,8 +12,6 @@ source $SCRIPT_DIR/defaults.sh
 cd $LOCAL_CHIPYARD_DIR
 
 # ignore the private vlsi submodules
-git config submodule.vlsi/hammer-cadence-plugins.update none
-git config submodule.vlsi/hammer-mentor-plugins.update none
 git config submodule.vlsi/hammer-synopsys-plugins.update none
```

Comment:
Did you mean to remove the cadence and synopsys commands and keep the mentor one?